### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by pgaskin**
I have fixed a few grammar problems in the issue template

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-1. Please speak English, this is the language everybody of us can speak and write.
-2. Please ask questions or config/deploy problems on our Gitter channel: https://gitter.im/go-gitea/gitea
-3. Please take a moment to search that an issue doesn't already exist.
-4. Please give all relevant information below for bug reports, incomplete details will be handled as an invalid report.
+1. Please speak English, this is the language all of us can speak and write.
+2. Please ask questions or configuration/deploy problems on our Gitter channel: https://gitter.im/go-gitea/gitea
+3. Please take a moment to search check that your issue doesn't already exist.
```
Comment (typeless) [inline .github/issue_template.md:3]: Combining 'check' and 'search' immediately without any punctuation looks suspicious to me (non-native speaker).

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by tboerger** [comment]
I think the other repos got a pretty similar issue template.

**Comment by pgaskin** [comment]
@tboerger Done

**Pull request merged**